charts/servicebusexporter/templates/deployment.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "servicebusexporter.fullname" . }}
+  labels:
+{{ include "servicebusexporter.labels" . | indent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "servicebusexporter.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "servicebusexporter.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+      {{- if .Values.addPromAnnotations }}
+      annotations:
+        prometheus.io/scrape: "true"
```

Comment:
Allowing any custom annotations may be reasonable. The `prometheus.io/*` annotations are the default for discovery, but it can also be based on other annotations depending on the config.